DataFormats/L1THGCal/interface/HGCalClusterT.h

```diff
@@ -107,7 +107,7 @@ namespace l1t
       
 
       double clusterXspread() const { 
-	if(cStep_ == ClusterStep::StepMulticluster && 1 == constituents_.size() ){
+	if(cStep_ == ClusterStep::SecondLayer && 1 == constituents_.size() ){
 	  return (HGCalClusterT(*(constituents_.begin()))).clusterXspread();
```

Comment:
I find these lines a bit strange and potentially dangerous.
1) I don't really understand the logic of deciding how the cluster spread is computed based on where it has been created. It should depend eventually of the type of cluster, but not on where it has been produced.
2) Having a call to `clusterXspread()` inside `clusterXspread()` could eventually create some infinite loop if we are not careful.